src/app/visit/status.hpp

```diff
@@ -0,0 +1,12 @@
+#pragma once
+#include <string>
+
+enum class Status
+{
+    Canceled,
+    Relocated,
+    Confirmed,
+    Visited
```

Comment:
Status could inherit from std::uint8_t 
and let's assign values here
Canceled = 1,
Relocated = 2...

Reply:
Done